### ISSUE TITLE
After registering wallet, there is no feedback - accessibility issue

### ISSUE THREAD

**Issue by ThinkBrandonThink** [openwallet-foundation-labs/learner-credential-wallet#536]
Clicking "Register" on the external website, there is no feedback on whether it was successful or not. This goes for both sighted users and screen reader users. The only thing that changes is that the white text in the button goes black and becomes unreadable. I recommend loading a success/error page that tells the user what happened. Doesn't need to be fancy, can just say "Wallet Registered! You may now close this page and return to your wallet." Or something like that. Similar language for if there was an error, but potentially add a button to try again or some text instructions on how to proceed.

Related work: Update register wallet on the LCW site, issue is here: https://github.com/digitalcredentials/learner-credential-wallet-site/issues/17

**Comment by alexfigtree** [comment]
Dmitri: In code of website, 1) display success message (in register function) and kickoff redirect back to wallet (show spinner), via deeplink.

**Comment by alexfigtree** [comment]
User should be redirected to https://lcw.app/mobile

**Comment by alexfigtree** [comment]
@kayaelle True, I will close this ticket.
